package.json

```diff
@@ -114,7 +114,7 @@
     },
     "scripts": {
         "start": "node server.js",
-        "start-nodemon": "nodemon server.js",
+        "start-nodemon": "node_modules/nodemon/bin/nodemon.js -L server.js",
```

Comment:
1. spell out path since symlinks trip up docker
2. need legacy mode `-L` inside docker

Reply:
I think the "correct" way to run things from inside `node_modules` is to use `npx nodemon ...`? This runs things out of `node_modules/.bin` (note `.`) which is where "scripts" get installed.

Reply:
Actually, commands in `scripts` inside `package.json` already have `node_modules/.bin` on the path, I believe, so we can just make this `nodemon ...`

Reply:
`/PrairieLearn/node_modules/.bin` is in the path, but `nodemon ...` doesn't work for some reason. I think my conclusion about symlinks was a red herring -- there's just something wrong with the way `nodemon` is packaged given how many "nodemon not recognized" google hits there are.

Reply:
`npx nodemon -L server.js` does work on my end.

Reply:
Seems like we could do better than `-L` but if the performance is good, I guess it doesn't matter?

Reply:
Seems specifically recommended for containers with mounted source: https://www.npmjs.com/package/nodemon#application-isnt-restarting